src/buildMathML.js

```diff
@@ -443,7 +499,20 @@ var buildGroup = function(group) {
  * we can do appropriate styling.
  */
 var buildMathML = function(tree, texExpression, settings) {
-    var expression = buildExpression(tree);
+    settings = settings || new Settings({});
+
+    var startStyle = Style.TEXT;
+    if (settings.displayMode) {
+        startStyle = Style.DISPLAY;
+    }
+
+    // Setup the default options
+    var options = new Options({
+        style: startStyle,
+        size: "size5"
+    });
```

Comment:
Since all of this is exactly the same as for `buildHTML`, and we don't modify `options`, do you want to pull this build into `buildTree` and pass down the `options` instead?